Creación de carpetas, rutas y controladores.

JulioF127 opened the pull request:
Se crearon las carpetas y archivos para las rutas y controladores de los roles para los usuarios.

JulioF127 committed:
Creación de carpetas, rutas y controladores

HJOC15 reviewed (changes requested):
Documentos vacíos.

DJPORTILLO30 reviewed (changes requested):
Faltan archivos de rutas y controladores

SFMZ21 reviewed (changes requested):
Agregar los archivos respectivos.

JulioF127 committed:
Corrección archivos vacíos

DJPORTILLO30 reviewed (changes requested):
cambiar el nombre  de los metodos a por ejemplo getRol, insertRol etc.

HJOC15 reviewed (changes requested):
Gracias por comentar el código. Hacer el cambio de mostarRoles a getRoles, por ejemplo, para mantener todo en misma sintonía con los demás. Todo lo demás correcto.

JulioF127 committed:
Modificacion de nombres de metodos

Se modificaron los nombres a idioma ingles.

DJPORTILLO30 reviewed (approved):
Todo en orden!

HJOC15 reviewed (approved):
¡Todo está bien!

JulioF127 committed:
Creación de Modelos y actualización de Rutas

Se creó el archivo que contiene los modelos y el controlador se adaptó para funcionar con este.

HJOC15 reviewed (approved):
¡Todo bien! ¡Buen trabajo!

SFMZ21 reviewed (approved):
Todo bien

DJPORTILLO30 reviewed (approved):
Todo okay
